app/src/comic_scrapers/pipelines.py

```
@@ -458,8 +640,7 @@ def _process_jp_comic_item(self, item: JpComicItem, spider):
             raise
         except Exception as e:
             spider.logger.error(
-                f"Failed to process JP Comic Item for"
-                f"{series_name_jp}, error: {str(e)}",
+                f"Failed to process JP Comic Item for{series_name_jp}, error: {str(e)}",
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)